apps/guide/src/content/03-topics/05-intents.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Intents
+category: Topics
+---
+
+# Intents
+
+Intents are an important part of establishing a WebSocket connection, as they define behavior regarding gateway events and impact received data via the REST API.
+
+## Usage
+
+```js
+import { Client, GatewayIntentBits } from 'discord.js';
+
+const client = new Client({
+	intents: [GatewayIntentBits.Guilds],
+});
+```
+
+This is the most basic usage of intents for discord.js. By specifying _`GatewayIntentBits.Guilds`_, your bot will receive gateway events regarding guilds. This includes receiving initial information about guilds it is in at startup, such as role data and the shard id to be used.
+
+You can find the full list of _`GatewayIntentBits`_ <DiscordAPITypesLink type="enum" parent="GatewayIntentBits">on the documentation</DiscordAPITypesLink> and an explanation of what each intent does [on Discord's API documentation](https://discord.com/developers/docs/topics/gateway#list-of-intents).
+
+## Considerations
+
+In discord.js, some intents require an extra bit of consideration.
+
+### _`GatewayIntentBits.Guilds`_
+
+discord.js relies heavily on caching in the library. We recommend you set at least the _`GatewayIntentBits.Guilds`_ intent to avoid these pitfalls.
+
+### _`GatewayIntentBits.GuildMembers`_
+
+Fetching members in a guild via <DocsLink type="class" parent="GuildMemberManager" symbol="fetch" brackets /> requests them over the gateway. As such, this intent is required and you may receive a timeout error if this intent is not specified.
+
+### _`GatewayIntentBits.DirectMessages`_
+
+This intent is required to receive direct messages. In discord.js however, you **must** specify partials as well. See the partials topic on how this is done.
```

Comment:
Let's give a direct link to the partials part where it specifies this 🙏

Reply:
This doesn't exist yet, remember?